src/routes/generator/StartModal.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+	import Modal from '$lib/Modal.svelte';
+	import { startModalOpen, files, imageLinkFromUser } from './stores';
+</script>
+
+<Modal open={true}>
+	<div class="container">
+		<div class="existingMaps">
+			<h3>View your existing maps</h3>
+		</div>
+		<div class="createNew">
+			<h3>Create from scratch</h3>
+			<div class="buttonContainer">
+				<input type="file" bind:files={$files} />
+			</div>
+			OR
+			<label style="margin-top:7%;background-color:transparent;font-weight:bold"
```

Comment:
Why does this have a background color of transparent? Isn't the label transparent by default?

Reply:
the label was white by default

Reply:
no it is not